# n_obj = -1 is waiting for image
# n_obj =  0 is not have object
# n_obj >= 1 is have a object
int8 n_obj
float64 cx
float64 cy
float64 area